truss/templates/control/supervisord.conf

```diff
@@ -0,0 +1,23 @@
+[supervisord]
+nodaemon=true
+logfile=/dev/null
+logfile_maxbytes=0
+
+
+[program:control-server]
+command=/control/.env/bin/python3 /control/control/server.py
+startsecs=0
+autostart=true
+autorestart=true
+stdout_logfile=/dev/fd/1
```

Comment:
This redirects the process's stdout to supervisor's stdout so that we can see it in the shell. Is this something we want here?

Reply:
We want to be able to pickup the control server logs from stdout for the UI